yoyodyne/models/pointer_generator.py

```diff
@@ -358,18 +358,75 @@ def forward(
                 h_features, c_features = self.init_hiddens(
                     len(batch), self.source_encoder.layers
                 )
-                predictions = self.decode(
-                    source_encoded,
-                    batch.source.mask,
-                    batch.source.padded,
-                    last_hiddens,
-                    self.teacher_forcing if self.training else False,
-                    features_enc=features_encoded,
-                    features_mask=batch.features.mask,
-                    target=batch.target.padded if batch.target else None,
-                )
+            predictions = self.decode(
```

Comment:
Is this a fix of sorts for #111?

Reply:
Yes. I should mark it in the PR description